fastapi_storages/base.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
In all places we are using `pathlib.Path` so please re-use that as it is a much nicer API.